pkg/controller/azvolumeattachment.go

```diff
@@ -684,17 +678,19 @@ func NewAzVolumeAttachmentController(ctx context.Context, mgr manager.Manager, a
 		return err
 	}
 
-	// call SyncAll regularly to keep replica count consistent
-	go func() {
-		for {
-			err := reconciler.SyncAll(ctx)
-			if err != nil {
-				klog.Warningf("%v", err)
-			}
-			time.Sleep(time.Duration(defaultSyncInterval) * time.Second)
-		}
-	}()
+	// initial sync
+	err = reconciler.SyncAll(ctx)
+	if err != nil {
+		klog.Warningf("failed to complete initial AzVolumeAttachment sync: %v", err)
+	}
 
 	klog.V(2).Info("AzVolumeAttachment Controller successfully initialized.")
 	return nil
 }
+
+func min(a, b int) int {
```

Comment:
Any reason we can't use Min in "math" package here?

Reply:
`Min` in math package uses float64 and requires conversion so decided to define one for the sake of simplicity :)